### DISCUSSION TITLE
Testnet status

### DISCUSSION THREAD

**Discussion by davy-vr** [gochain-io/docs#149] [Q&A]
Hi,

I have a question regarding the testnet status. I am following the docs from https://github.com/gochain/docs/wiki/testnet but both the rpc as the testnet explorer seem to be unavailable.

Have they moved or is there an issue?

Thank you

**Comment by treeder** [comment]
Hi, we shutdown the testnet as it wasn't being used enough to justify it. Mainnet transactions are cheap enough to test on (fraction of a penny typically) so we recommend just using mainnet to test on.

**Reply by ghost** [reply]
Dear treeder and GoChain Team,

I hope this message finds you well.

I recently learned that the GoChain Testnet has been shut down due to low usage. While I understand the decision given the circumstances, I wanted to share some thoughts on how a dedicated testnet could be crucial for maintaining the high standards and reputation of the GoChain ecosystem.

Observations and Feedback:

Importance of a Dedicated Testnet: Even though the GoChain Mainnet offers affordable transaction fees, a dedicated Testnet is vital for developers to test and refine their projects. This allows for thorough development and quality assurance before deploying on the mainnet. Without a dedicated testnet, developers might face challenges in ensuring the robustness of their implementations.

Impact on GoChain’s Reputation: The absence of a GoChain Testnet could lead developers to use alternative testnets, which might not accurately reflect the GoChain Mainnet environment. This shift could result in a higher likelihood of poorly tested or substandard projects being deployed on GoChain. Consequently, the Mainnet might become associated with numerous low-quality or inadequately tested projects, impacting its reputation.

Quality Assurance and Ecosystem Integrity: A dedicated testnet helps ensure that only high-quality, well-tested projects are launched on the GoChain Mainnet. It provides a platform for rigorous testing and iterative development, which helps in maintaining the integrity and reputation of the GoChain ecosystem. Without it, there is a risk of an influx of poorly vetted projects, which could undermine the perceived value and reliability of GoChain as a blockchain platform.

Community and Developer Engagement: Reinstating or enhancing the GoChain Testnet could also foster greater community engagement and support. It demonstrates GoChain’s commitment to quality and provides developers with the tools they need to build robust and reliable projects.

Thank you for considering this feedback. We believe that a dedicated testnet could play a crucial role in maintaining GoChain’s reputation for quality and reliability.

I look forward to your thoughts and would be happy to discuss this further if needed.

Best regards,

**Reply by treeder** [reply]
Hi @SpaceEmperor , I appreciate your comments and concerns, but is there any reason you can't test on mainnet?
